pages/release-notes.mdx

```diff
@@ -70,17 +70,37 @@ updated.
 
 - 
 
-{<h4>✨ New features</h4>}
+{<h4>✨ New features and improvements</h4>}
 
 - 
 
-{<h4>🛠️ Improvements</h4>}
+{<h4>🐞 Bug fixes</h4>}
 
-- 
+-
+
+#### Lab v2.19.0 - Nov 27, 2024
+
+{<h4>✨ New features and improvements</h4>}
+
+- The Lab web application can now be started on a
+  custom base path using the environment variable
+  `BASE_PATH`, which is useful for reverse proxy
+  configurations.
+- In multi-tenant environments with many databases,
```

Comment:
Is "with many databases" part necessary?
I would go with:

> In a multi-tenant environment, a search bar is added to filter databases by name.

Reply:
Makes sense! Will fix it.